app/models/news_search.rb

```diff
@@ -63,7 +63,11 @@ def results_to_hash
 
   # TO-DO: Normalize news results in https://cm-jira.usa.gov/browse/SRCH-4071
   def normalized_results
-    @results
+    {
```

Comment:
This is all temporary/placeholder code that will be changed or moved elsewhere once we integrate news searches in SRCH-4071, so this can be ignored.